source/simulate/analyse-simulation.md

```diff
@@ -1,16 +1,16 @@
 # Analysing or debugging a simulation
 
-When simulating with the Python API, you might want to understand how the result was calculated and what intermediate variables and parameters have been taken into account.
+To understand how a result was calculated when simulating with the Python API, intermediate variables and parameters have been taken into account.
```

Comment:
Gone with "analysis of the intermediate variables and parameters have been taken into account." to appease my need to avoid the use of the word "you".

Reply:
Gone with "analysis of the intermediate variables and parameters have been taken into account." to appease my need to avoid the use of the word "you".